examples/hotrod/pkg/tracing/init.go

```diff
@@ -70,9 +90,8 @@ func Init(serviceName string, exporterType string, metricsFactory metrics.Factor
 			semconv.ServiceNameKey.String(serviceName),
 		)),
 	)
-	otTracer, _ := otbridge.NewTracerPair(tp.Tracer(""))
-	logger.Bg().Debug("created OTEL->OT bridge", zap.String("service-name", serviceName))
-	return otTracer
+	otTracer, _ := otbridge.NewTracerPair(tp.Tracer(serviceName))
```

Comment:
I think now we've established that you can mix OT and OTEL tracers for a single trace. 

Given the above, I wonfer if we should remove the otebridge dependency altogether.

Reply:
@albertteoh we cannot remove the bridge right now, because only `redis.go` was changed to use OTEL instrumentation. My recommendation is to merge this PR when clean, then apply the same principles to all other places where OT instrumentation is still used. Only then we can remove the bridge dependency.

Reply:
And yes, it was great to see that OT and OTEL instrumentations were able to interoperate when using the bridge.